rpcperms/interceptor.go

```diff
@@ -120,6 +188,14 @@ func (r *RPCInterceptor) CreateServerOpts() []grpc.ServerOption {
 		strmInterceptors, errorLogStreamServerInterceptor(r.rpcsLog),
 	)
 
+	// Add RPC state check interceptors.
+	unaryInterceptors = append(
+		unaryInterceptors, r.rpcStateUnaryServerInterceptor(),
+	)
+	strmInterceptors = append(
+		strmInterceptors, r.rpcStateStreamServerInterceptor(),
+	)
+
 	// If any interceptors have been set up, add them to the server options.
 	var serverOpts []grpc.ServerOption
 	if len(unaryInterceptors) != 0 && len(strmInterceptors) != 0 {
```

Comment:
We know none of them are empty as we've just added a bunch above.

Reply:
Haha, true! Did just copy this, but you're right this if is not needed anymore 👍